src/javascript/app/pages/user/metatrader/metatrader.config.js

```diff
@@ -122,7 +122,7 @@ const MetaTraderConfig = (() => {
 
             if (!Client.get('currency')) {
                 resolve($messages.find('#msg_set_currency').html());
-            } else if (is_virtual && !accounts_info[acc_type].is_demo) { // virtual clients can only open demo MT accounts
+            } else if (is_virtual && Client.hasOnlyVirtualAccounts()) { // virtual clients without a real account can only open demo MT accounts
```

Comment:
why can't we use the existing function `hasAccountType('real')`?

Reply:
@negarn You're right.